examples/toolbar/help.html

```diff
@@ -19,7 +19,7 @@
 
 <h1>Toolbar Help</h1>
 
-<p>Learn more about toolbar behaviors by reading the <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#toolbar">ARIA Authoring Practices Guide</a>.</p>
+<p>Learn more about toolbar behaviors by reading the <a href="https://w3c.github.io/aria-practices/#toolbar">ARIA Authoring Practices Guide</a>.</p>
```

Comment:
Should we change this to a relative URL? Not sure why it wasn't to begin with tbh.

Reply:
good point! pushed a fix for that.